apps/schema/static/schema/swagger.yaml

```diff
@@ -176,6 +176,56 @@ paths:
               allOf:
               - $ref: '#/components/requestBodies/shipment/createAttributes'
 
+  /api/v1/shipments/{shipment_id}/tracking/:
+    post:
+      summary: Add tracking data
+      description: >
+        Get additional info about a `Shipment`.
+      parameters:
+      - $ref: '#/components/parameters/shipment/path'
+      tags:
+      - Shipments
+      responses:
+        '204':
+          description: *response_204
+        '401':
+          description: *response_401
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: '#/components/schemas/errors/401'
+      requestBody:
+        content:
+          application/vnd.api+json:
+            schema:
+              $ref: '#/components/requestBodies/shipment/tracking/createRequest'
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/requestBodies/shipment/createAttributes'
+          multipart/form-data:
+            schema:
+              allOf:
+              - $ref: '#/components/requestBodies/shipment/createAttributes'
+
+    delete:
```

Comment:
There should not be a `delete` tracking data endpoint. Remove this and make sure `delete` does not exist on that endpoint.